pages/public_cloud/containers_orchestration/managed-rancher-service/getting-started/guide.en-gb.md

```diff
@@ -0,0 +1,276 @@
+
+4. Click on **Get Project List** and select the **Project** were you want to create your cluster.\
+5. Click on **Continue**.
+
+6. Set your **Cluster Name** and **Cluster Decription**.\
+![Cluster Creation](images/PublicCloudDriver_creation_cloud_credential.png)
+7. Create a **Machine Pool** for each Kubernetes role. Refer to the [Best Practices](https://ranchermanager.docs.rancher.com/how-to-guides/new-user-guides/launch-kubernetes-with-rancher/use-new-nodes-in-an-infra-provider#node-roles), [Node Requirements for Rancher Managed Clusters](https://ranchermanager.docs.rancher.com/how-to-guides/new-user-guides/kubernetes-clusters-in-rancher-setup/node-requirements-for-rancher-managed-clusters) and [Recommended Cluster Architecture](https://ranchermanager.docs.rancher.com/how-to-guides/new-user-guides/kubernetes-clusters-in-rancher-setup/checklist-for-production-ready-clusters/recommended-cluster-architecture) guides for recommendations on role assignments and counts.
+![Cluster Creation](images/PublicCloudDriver_creation_machine_pool.png)
+    For each machine pool, define the machine configuration by providing the following parameters:
+    - **Pool Name** - Name of the Machine Pool
+    - **Machine Count** - Number of instances
+    - **Roles** - `etcd`, `Control Plane` or `Worker`. You can refer to the [Node Roles](https://ranchermanager.docs.rancher.com/how-to-guides/new-user-guides/launch-kubernetes-with-rancher/use-new-nodes-in-an-infra-provider#node-roles) documentation.
+    - **Region** - The OVHcloud Public Cloud region. If you want to check the availability of specific products that you plan to use alongside Kubernetes, you can refer to the [Availability of Public Cloud Product](https://www.ovhcloud.com/en/public-cloud/regions-availability/) page.
+    - **Flavor** - The instance flavor used for your nodes. You can refer to the [OVHcloud Flavor list](https://www.ovhcloud.com/en/public-cloud/prices/#13569).
+    - **Image** - The Operating System image used for your nodes. Please refer to [Rancher Operating Systems and Container Runtime Requirements](https://ranchermanager.docs.rancher.com/how-to-guides/new-user-guides/kubernetes-clusters-in-rancher-setup/node-requirements-for-rancher-managed-clusters).
+    - **Key Pair** - The SSH Key Pair used to access your nodes. Please refer to this guide on [how to create a SSH KeyPair and add it to your Public Cloud project](https://help.ovhcloud.com/csm/en-public-cloud-compute-getting-started?id=kb_article_view&sysparm_article=KB0051009).
+    - **Security Group** - Existing security groups from our Public Cloud project
+    - **Availability Zone** - Only `nova` is supported at the moment.
+    - **Floating IP Pools** - Only `Ext-Net` is supported at the moment.
+    - **Networks** - Public Cloud Network to attach your nodes
```

Comment:
Note that users will also need a gateway to make the driver working.

Reply:
I probably need to add a 'prerequisites' section then